ui/dashboards/src/components/Panel/PanelLinks.tsx

```diff
@@ -0,0 +1,114 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { IconButton, Link as LinkComponent, ListItemIcon, ListItemText, Menu, MenuItem, Theme } from '@mui/material';
+import LaunchIcon from 'mdi-material-ui/Launch';
+import { Link } from '@perses-dev/core';
+import { MouseEvent, useState } from 'react';
+import { InfoTooltip } from '@perses-dev/components';
+import { useReplaceVariablesInString } from '@perses-dev/plugin-system';
+
+export function PanelLinks({ links }: { links: Link[] }) {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const isMenuOpened = Boolean(anchorEl);
+  const handleOpenMenu = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  // If there is only one link, show it directly
+  if (links.length === 1 && links[0]) {
+    const link = links[0];
+    return <LinkButton link={link} />;
+  }
+
+  // Else we show a menu with a list of all links
+  return (
+    <>
+      <InfoTooltip description={`${links.length} links`} enterDelay={100}>
+        <IconButton
+          aria-label="Panel links"
+          size="small"
+          onClick={handleOpenMenu}
+          sx={(theme) => ({ borderRadius: theme.shape.borderRadius, padding: '4px' })}
+        >
+          <LaunchIcon
+            aria-describedby="links-icon"
+            fontSize="inherit"
+            sx={{ color: (theme) => theme.palette.text.secondary }}
+          />
+        </IconButton>
+      </InfoTooltip>
+
+      <Menu
+        anchorEl={anchorEl}
+        open={isMenuOpened}
+        onClose={handleClose}
+        MenuListProps={{
+          'aria-labelledby': 'panel-links',
+        }}
+      >
+        {links.map((link: Link) => (
+          <LinkMenuItem key={link.url} link={link} />
+        ))}
+      </Menu>
+    </>
+  );
+}
+
+function LinkButton({ link }: { link: Link }) {
+  const { url, name, tooltip, targetBlank } = useLink(link);
+
+  return (
+    <InfoTooltip description={tooltip ?? url} enterDelay={100}>
+      <IconButton
+        aria-label={name ?? url}
+        size="small"
+        href={url}
+        target={targetBlank ? '_blank' : '_self'}
+        sx={(theme) => ({ borderRadius: theme.shape.borderRadius, padding: '4px' })}
+      >
+        <LaunchIcon fontSize="inherit" sx={{ color: (theme: Theme) => theme.palette.text.secondary }} />
+      </IconButton>
+    </InfoTooltip>
+  );
+}
+
+function LinkMenuItem({ link }: { link: Link }) {
+  const { url, name, tooltip, targetBlank } = useLink(link);
+
+  return (
+    <InfoTooltip description={tooltip ?? url} enterDelay={100}>
+      <MenuItem component={LinkComponent} href={url} target={targetBlank ? '_blank' : '_self'}>
+        <ListItemIcon>
```

Comment:
Imho repeating this icon on each item is useless since it's always the same
![image](https://github.com/perses/perses/assets/7058693/7839312e-6d38-4439-8fe6-c3496e76982e)
(+ considering we can still see the links button with the icon when the dropdown is opened)

Reply:
I remember we had a similar discussion for the projects dropdown and we ended up removing the icons:
![image](https://github.com/perses/perses/assets/7058693/01028649-d2f3-4cce-a261-a42d50964d24)